packages/transformer/src/IModelTransformer.ts

```diff
@@ -309,6 +316,7 @@ export class IModelTransformer extends IModelExportHandler {
     }
     this.sourceDb = this.exporter.sourceDb;
     this.exporter.registerHandler(this);
+    this.exporter.setExportElementAspectsStrategy(this._options.elementAspectExportStrategy ?? ElementAspectExportStrategy.WithElement);
```

Comment:
I would prefer we allow people passing in a custom exporter to set this themselves, and we can just log a warning (or throw an error) if they pass in a custom exporter at the same time as passing not "undefined" to the transformer options.

Reply:
Yeah, that looks better. It allowed to remove the new option from `IModelTransformer`, remove aspect export strategies enum and `setExportElementAspectsStrategy` from the API of `IModelExporter`.